platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/config/PathsConfig.java

```diff
@@ -25,21 +25,21 @@
 
 /**
  * Configurations related to paths.
- * @param settingsUsedDir
- *     the directory where the settings used file will be created on startup if and only if settings.txt exists
- * @param keysDirPath
- *     path to data/keys/
- * @param appsDirPath
- *     path to data/apps/
- * @param logPath
- * 	   path to log4j2.xml (which might not exist)
+ *
+ * @param settingsUsedDir the directory where the settings used file will be created on startup if and only if
+ *                        settings.txt exists
+ * @param keysDirPath     path to data/keys/
+ * @param appsDirPath     path to data/apps/
+ * @param logPath         path to log4j2.xml (which might not exist)
+ * @param markerFilesDir  path to the directory where marker files are written.
  */
 @ConfigData("paths")
 public record PathsConfig(
         @ConfigProperty(defaultValue = ".") String settingsUsedDir,
         @ConfigProperty(defaultValue = "data/keys") String keysDirPath,
         @ConfigProperty(defaultValue = "data/apps") String appsDirPath,
-        @ConfigProperty(defaultValue = "log4j2.xml") String logPath) {
+        @ConfigProperty(defaultValue = "log4j2.xml") String logPath,
+        @ConfigProperty(defaultValue = "none") String markerFilesDir) {
```

Comment:
Would it make sense to set this to have a default value of `/dev/null` and to drop support for the the `none` keyword?

Reply:
I've added a flag for turning on the feature, default value is off, and have a valid location set for the default directory.  I've dropped the `none` option.

Reply:
The location of the boolean feature flag may need to be moved to a different config, I don't know which config it belongs in.

Reply:
I'm ok with it here for now.